src/components/editor/ImportResourceTemplate.jsx

```diff
@@ -121,6 +145,9 @@ class ImportResourceTemplate extends Component {
       // The 0 is the profileCount which is only used for tracking create errors
```

Comment:
ok, "profileCount" confuses me here, and "create errors" too -- the errors when ... something ... is created?  what?

Reply:
The `profileCount` stores a count of the RTs in the uploaded file, and `createResourceErrors` keeps track of errors that might happen while loading individuals RTs within a profile. I don't think I've touched related code in this branch, so not too inclined to make modifications around this.

Reply:
fair enough.  So curiously, "profileCount" is really "resourceTemplateCount" and "createResourceErrors" is "resourceTemplateLoadErrors".  K.